# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/res/shared.glsl
# Commit: 86db847cfe57
# Full Hash: 86db847cfe570dba25289ffbe52aaaaecf76bc70
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-18 04:07:40
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r c3495de132cd -r 86db847cfe57 gfx/wr/webrender/res/shared.glsl
--- a/gfx/wr/webrender/res/shared.glsl	Thu Dec 17 19:53:54 2020 +0000
+++ b/gfx/wr/webrender/res/shared.glsl	Thu Dec 17 19:54:02 2020 +0000
@@ -163,7 +163,7 @@
 uniform samplerExternalOES sColor0;
 uniform samplerExternalOES sColor1;
 uniform samplerExternalOES sColor2;
-#else
+#elif defined WR_FEATURE_TEXTURE_2D_ARRAY
 uniform sampler2DArray sColor0;
 uniform sampler2DArray sColor1;
 uniform sampler2DArray sColor2;